nservicebus/rabbitmq/routing-topology_support-delayed-delivery_rabbit_[4,].partial.md

```diff
@@ -0,0 +1,7 @@
+### Delayed delivery
+
+Starting with Version 4.3, it is possible to support [delayed delivery](delayed-delivery.md) without requiring the [timeout manager](/nservicebus/messaging/timeout-manager.md) by implementing `ISupportDelayedDelivery` in addition to `IRoutingTopology` in the routing topology class.
+
+When the routing topology implements `ISupportDelayedDelivery`, the transport will call the `BindToDelayInfrastructure` method to allow the routing topology to create the appropriate bindings to the delivery exchange that is part of the native delayed delivery infrastructure. 
+
+`IRoutingTopology` and `ISupportDelayedDelivery` will be combined into a single interface in Version 5.0.
```

Comment:
> We should add something in https://docs.particular.net/nservicebus/messaging/timeout-manager, there we say

>>NServiceBus provides a delayed-delivery implementation for transports that don't support it natively.

>which doesn't seem to be technically correct?

I'm not seeing what's incorrect about that. If a transport doesn't have some sort of native support for delayed delivery, like we're adding for the RabbitMQ transport, then it does have to use the timeout manager.

> Also https://docs.particular.net/nservicebus/messaging/delayed-delivery#how-it-works

> One thing could be simply adding a sentence, saying that Rabbit in Versions x and below uses TM, in versions X and above has native delayed delivery implementation + link to the new article with details

I'll tweak the wording there as well.